pycbc/filter/qtransform.py

```diff
@@ -35,10 +35,72 @@
 from math import pi
 import numpy as np
 from pycbc.strain  import next_power_of_2
+from pycbc.types.timeseries import FrequencySeries, TimeSeries
+from numpy import fft as npfft
 
 __author__ = 'Hunter Gabbard <hunter.gabbard@ligo.org>'
 __credits__ = 'Duncan Macleod <duncan.macleod@ligo.org>'
 
+def qtransform(data, Q, f0, sampling, normalized):
```

Comment:
So, what I wrote makes it a keyword argument. I think you are thinking of functions that accept *arbitrary* keyword arguments as follows. We don't want to do that.

def f(**kwds)